lib/jsdom/living/nodes/HTMLButtonElement-impl.js

```diff
@@ -47,6 +49,60 @@ class HTMLButtonElementImpl extends HTMLElementImpl {
         break;
     }
   }
+
+  get willValidate() {
+    return !isDisabled(this);
```

Comment:
This looks quite different from the [spec](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#candidate-for-constraint-validation).